app/controllers/api/v1/authors_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Api
+  module V1
+    class AuthorsController < ApplicationController
+      before_action :set_author, only: [:show, :update, :destroy]
```

Comment:
Don't use this, you should never use callbacks for setting instance variables (rather use it for controlling the app flow, such as redirecting users if they don't have the right permissions)